### ISSUE TITLE
Single orientation asv in mixed orientation amplicon library?

### ISSUE THREAD

**Issue by DollyPrrrton** [benjjneb/dada2#1880]
Hi Benjamin,
Thanks for all your work on dada2 and this page.

I’ve come across something in a recent library that I havent been able to figure out from a few days exploring possibilities and trawling through the discussions here. I thought I would post it here and see if any one had any experience or knowledge of similar. Apologies if ive missed something very obvious. Im still getting my head round some aspects of metabarcoding and dada2.

The library is a small test library (Miseq Nano run) to check primers and barcoding effectiveness. It’s a mixed orientation amplicon library prepared by ligation. For almost all asv’s im therefore getting two asv counts, forward and reverse, at around 50/50 ratio. What’s bothering me is one of the ASV’s, which happens to be a taxa I am particularly interested in capturing diversity in, doesn’t have a reverse counterpart. Im therefore trying to figure out if the asv is a pcr or seq artifact, a processing error, or a dropout of the reverse counterpart, and whether I should remove it from the downstream analysis?

A few more details and avenues ive looked at:
-	The asv makes up around 3% of the filtered library read count. 
-	Its seen across multiple samples within the library, that were amplified with different barcoding indexes and across multiple pcr reactions
-	It contains a unique snp for this taxa so doesn’t look like a chimera of other asvs – its found in the first half of forward orientation R1 read, so not covered by the overlapping read-merge
-	It is found consistently across multiple processing approaches in dada2 (mixed-oriented amplicons together, or split across two runs, varying filtering and demultiplexing parameters, pseudo-pooling)
-	Unlikely to be a barcoding bias as the bias for other asv’s is generally towards the missing asv orientation here (barcoding bias is a separate issue I need to resolve..)
-	I cant think of a biological scenario where this might occur. PCR amplification should amplify both strands after initial cycles regardless of which strand this variant sat on?

Im scratching my head here. Any help or insight would be greatly appreciated. At the end of the day I can remove the asv from further analysis if I cant figure it out, but would like to understand it if possible.

Many thanks

**Comment by benjjneb** [comment]
That does sound strange and worthy of follow-up.

I would start by trying to nail down exactly where the observed variant and expected reverse-complement variant fail to both appear in the data.

Since the discriminating bases are in the forward-orientation R1 part of the read, it should be possible to create the observed R1 sequence, and the expected (but missing) R2 sequence from the reverse complement orientation.

Beginning at the beginning: Are those sequences present in the raw sequence data? (maybe using a sample where the observed ASV was seen at relatively high abundance).

**Comment by DollyPrrrton** [comment]
Hi Benjamin,

Thanks very much for the response. That was helpful. I went and checked back one of the samples raw files. The snp of interest was located at the forward orientation 5' end, so should have been present in Forw_Orientation.R1 and Rev_Orientation.R2. I checked with blast and grep -c and it was, at fairly even counts, and around the same count as the forward ASV output in the original run. (As an aside, can this approach be used as a "belts-and-braces" approach to validate ASVs of interest?)

Ive run out of time now, but will check a few other steps next. If its present in the raw Rev.orientation.R2 then it should have merged and been present as a the reverse orientation ASV. Im wondering if its been removed by removeBimeraDenovo? I'll try and find time next week to check. 

Thanks again for the help!
